figures/board-example.txt

```diff
@@ -0,0 +1,30 @@
+Editors' Note: add the board example in slide 4

```

Comment:
remove the notes here. Only the pictures are needed.

Reply:
just kept to simplify the review: to be removed before merging the PR

Reply:
Should be fixed with 77c6449bb96fc1fcdc3e975e1ee17d0a5bc07a34